scss/components/_activity-panel.scss

```diff
@@ -194,5 +197,43 @@
       background-color: $brand-danger !important;
     }
   }
+
+  .crm-form-date-wrapper {
+    display: inline-block;
+  }
+
+  // This special width is added to accomodate date picker icon
+  // Some how the final width applied is 168 - 42 to the input element
+  // and 34px to date picker icon = which counts to: 160px.
+  .crm-form-date {
+    width: $civicase-activity-panel-form-date-item-width + (42px - 34px);
+  }
+
+  // Remaining space (230 - 160px) - 70 - 10px of margin.
+  .crm-form-time {
+    margin-left: 10px;
+    width: $civicase-activity-panel-form-item-width - $civicase-activity-panel-form-date-item-width - 10px;
```

Comment:
The comment says `(230 - 160px) - 70 - 10px`. But looks like in the calculation 70px is missing, is the comment correct?

Reply:
Oops. It should be `=70` rather.